libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -43,6 +43,10 @@ smt::SortPointer dev::solidity::smtSort(Type const& _type)
 		smt::SortPointer returnSort = smtSort(*returnTypes.at(0));
 		return make_shared<smt::FunctionSort>(parameterSorts, returnSort);
 	}
+	case smt::Kind::Array:
+	{
+		solAssert(false, "Invalid type");
+	}
```

Comment:
Will be implemented in the PR that adds support to `mapping`.

Reply:
`solUnimplementedAssert` in that case